ingestion-edge/docker-compose.circleci.yml

```diff
@@ -4,6 +4,23 @@ version: '3'
 
 services:
   test:
-    entrypoint: docker/pytest-upload-coverage
+    command:
+    - --black
+    - --flake8
+    - --docstyle
+    - --mypy
+    - --mypy-ignore-missing-imports
+    - --ignore=tests/load
+    - .
+    - tests/integration
+    - --server=http://web:8000
```

Comment:
entrypoint is gone because we no longer upload coverage from here

Reply:
I'm a bit worried that these pytest configuration flags are duplicated in many places now, so seems like there's a good likelihood they'll drift apart from one another. Could we pull most of this configuration into a `pytest.ini` or wrapper of some sort that will be consumed in both the docker-compose case and the circleci case?

Reply:
pulled the pytest flags into `bin/test` wrapper